modules/cid_checker/output.tf

```diff
@@ -0,0 +1,13 @@
+
+output "s3_bucket_id" {
+  value = aws_s3_bucket.cid_checker_market_deals.id
+}
+
+output "iam_role_name" {
+  value = aws_iam_role.cid_checker_sync_market_deals.name
+}
+
+output "iam_role_policy_arn" {
+  value = aws_iam_policy.cid_checker_sync_market_deals.arn
+}
+
```

Comment:
what is the goal of keeping output resources if we don't use them in our code?

Reply:
Removed outputs.tf from module. Resources on this module use only inside module.